### PR TITLE
also run jshint and jscs on tests

### PR DESCRIPTION
**Pull request by fippo**
jshint and jscs were not run on test/test.js

### CODE REVIEW

File: test/test.js [hunk 1]
```diff
@@ -159,9 +161,9 @@ test('basic connection establishment with promise', function(t) {
     if (event.candidate) {
       var cand = new RTCIceCandidate(event.candidate);
       pc.addIceCandidate(cand)
-          .then(function () {
+          .then(function() {
           })
-          .catch(function (err) {
+          .catch(function(err) {
```
Comment (alvestrand) [inline test/test.js]: Should we match these on style with the case below, and wrap .then and .catch up to the previous line?

File: test/test.js [hunk 2]
```diff
@@ -173,39 +175,39 @@ test('basic connection establishment with promise', function(t) {
...
     devices.forEach(function(d) {
-      t.ok(d.kind == 'videoinput' ||
-           d.kind == 'audioinput' ||
-           d.kind == 'audiooutput', 'Known device kind');
+      t.ok(d.kind === 'videoinput' ||
+           d.kind === 'audioinput' ||
+           d.kind === 'audiooutput', 'Known device kind');
       t.ok(d.deviceId.length !== undefined, 'device id present');
       t.ok(d.label.length !== undefined, 'device label present');
     });
```
Comment (alvestrand) [inline test/test.js:210]: Didn't catch this when I first looked at it, but aren't we missing a ".catch(function (err) { t.fail(err.toString()); }) here?

File: test/test.js (hunk 1 above)
Comment (fippo) [inline test/test.js]: removed the empty success callback (consistent with what the spec example shows) and moved up the catch to the previous line.

File: test/test.js (hunk 2 above)
Comment (jan-ivar) [inline test/test.js:210]: No, it's there. The code just gets cut off here. ;) (click the "unfold" symbol to see it)

**Pull request merged**